test/fixtures/check.angular.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
https://github.com/prettier/prettier/blob/ce26805e84756953875f30a0c165bd6b09c94ea0/src/language-html/index.js#L13 suggests that you can trigger the angular parser by using the `.component.html` extension. 

If we name this file `check.inert.component.html` then it looks like we should trigger using the angular component without needing to add any of the overrides in our base prettierrc.